src/bundle/Resources/public/js/scripts/fieldType/base/base-rich-text.js

```diff
@@ -220,6 +222,44 @@
 
             return alloyEditor;
         }
+
+        countWordsCharacters(container, editorHtml) {
+            let counterWrapper = container.parentElement.getElementsByClassName('ez-character-counter');
+
+            if (counterWrapper.length) {
+                let wordCharacterWrappers = counterWrapper[0].getElementsByTagName('span');
+                let characterCount = 0;
+                let wordCount = 0;
+                let countableTags = this.fetchCountableTags(editorHtml);
```

Comment:
`const`

Reply:
`characterCount` and `wordCount` are modified right after, they should also be set as `const`?

Reply:
No, you will not be able to assign a new value to these variables.